src/stars.css

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child ,
+.stars--2 .stars__star:nth-child(-n+2) ,
+.stars--3 .stars__star:nth-child(-n+3) ,
+.stars--4 .stars__star:nth-child(-n+4) ,
+.stars--5 .stars__star:nth-child(n) {
```

better to use consistent selectors
```suggestion
.stars--1 .stars__star:nth-child(-n+1) ,
.stars--2 .stars__star:nth-child(-n+2) ,
.stars--3 .stars__star:nth-child(-n+3) ,
.stars--4 .stars__star:nth-child(-n+4) ,
.stars--5 .stars__star:nth-child(-n+5) {